bundle/edu.gemini.spModel.io/src/test/scala/edu/gemini/spModel/io/impl/migration/to2016A/PlutoDemotionTest.scala

```diff
@@ -28,7 +30,7 @@ class PlutoDemotionTest extends MigrationTest {
             e.toList match {
               case List((t, c)) =>
 
-                  Assert.assertEquals("ValidAt", "10/28/15 7:39:58 PM UTC", SPTargetPio.formatter.format(t))
+                  Assert.assertEquals("ValidAt", "10/28/15 7:39:58 PM UTC", SPTargetPio.formatDate(new Date(t)))
```

Comment:
If this is the only place we are using it perhaps we should just move it to test, or is it used internally in `SPTargetPio`? If so, then all calls should go through `formatDate`, shouldn't they?

Reply:
Good question.  It's used internally in `SPTargetPio`.  It was `public` before and that's how it was visible to the test case.  Inside of `SPTargetPio` it is only used for parsing dates and that usage already was synchronized via the `parseDate` method.  The only usage for formatting was in the test case and now that has been synchronized as well so that it cannot be used simultaneously with parsing.

Reply:
Thanks for the clarification